conf/application.conf

```diff
@@ -96,7 +96,7 @@ microservice {
     hawk {
         protocol = http
         host = localhost
-        port = 10903
+        port = 10908
```

Comment:
Just double checking. Is pointing to the Hawk stub in local, dev and stage environment right? Because the Api test will fail. Does QA and UI know these changes are coming?

Reply:
oh yes, I will change this back to 10903